perun-core/src/main/java/cz/metacentrum/perun/core/api/ServicesManager.java

```diff
@@ -380,30 +380,24 @@
 	 *
 	 * attributes: {...hashes...}
 	 * hierarchy: {
-	 *    ** facility **
-	 *    hashes: [...hashes...]
-	 *    members: []
-	 *    children: [
-	 *      {
-	 *        ** resource1 **
-	 *        hashes: [...hashes...]
-	 *        children: []
-	 *        members: [
-	 *          {
-	 *            ** member 1 **
-	 *            hashes: [...hashes...]
-	 *          },
-	 *          {
-	 *            ** member 2 **
-	 *            ...
-	 *          }
-	 *        ]
-	 *      },
-	 *      {
-	 *        ** resource2 **
-	 *        ...
-	 *      }
-	 *    ]
+	 *   "1": {    ** facility id **
+	 *     members: {    ** all members on resource **
+	 *        "4" : 5,    ** member id : user id **
+	 *        "6" : 7,    ** member id : user id **
+	 *       ...
+	 *     }
+	 *     children: [
+	 *       "2": {    ** resource id **
+	 *         children: []
+	 *         members: {
```

Comment:
These should be all members on the resource 2.

Reply:
I have added a comment in the javadocs.